packages/fast-components-react-msft/src/auto-suggest-option/auto-suggest-option.tsx

```diff
@@ -100,8 +99,8 @@ class AutoSuggestOption extends Foundation<
         searchString: string
     ): React.ReactNode => {
         if (
-            !isNullOrUndefined(displayString) &&
-            !isNullOrUndefined(searchString) &&
+            !isNil(displayString) &&
+            !isNil(searchString) &&
```

Comment:
While this does need to change, it is not related to this PR. Your PR title is about a stylistic fix; this code change here is in relation to fixing an import statement to not rely on nodeJs internal utilities. I would prefer not to conflate these two as the same issue. It's easy enough to fix this in a different PR.

Reply:
Then my suggestion is, we abandon this PR. First thing, is to create a new PR that is fixes the import issue. Then once the import fix is committed, tackle the style change. Will that work?

Reply:
Alternatively; I think the issue you are solving here is not the import. Is there a reason to not just remove that from this branch and do that in a different PR? It seems like that way you don't have to abandon the PR and instead you can just move forward with both.

Reply:
The PR is now only the margin fixes in action trigger component